sharing/resource_share.go

```diff
@@ -208,17 +206,28 @@ func ResourceShare() common.Resource {
 			return nil
 		},
 		Read: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
-			si, err := NewSharesAPI(ctx, c).get(d.Id())
```

Comment:
si.sortSharesByName()
	si.suppressCDFEnabledDiff()

would also be needed here as we were doing it before + would be required before we do StructToData below.

Reply:
Addressed in [910decf](https://github.com/databricks/terraform-provider-databricks/pull/3916/commits/910decfab1c52c5f522d3c5e69714aaf1d714bed)